src/containers/orgs/OrgMemberContainer.js

```diff
@@ -69,9 +68,9 @@ const OrgMemberContainer = ({ member, organization, organizationId } :Props) =>
   const orgPath = Routes.ORG.replace(Routes.ORG_ID_PARAM, organizationId);
 
   const userProfileLabel = getUserProfileLabel(member);
-  const userId = getUserId(member);
+  // const userId = getUserId(member);
```

Comment:
do you need this comment anymore?

Reply:
Nope. Actually, this component is not used anywhere. I've removed it in another branch.